cache/class-vary-cache.php

```diff
@@ -335,7 +335,7 @@ public static function is_encryption_enabled() {
 	 * @throws string If credentials ENV Variables aren't defined.
 	 * @return string encrypted version of string
 	 */
-	private static function encrypt_cookie_value( $value ) {
+	public static function encrypt_cookie_value( $value ) {
```

Comment:
What's the reason for switching this to `public`?

Reply:
Whoops, that was a left-over artifact from calling it to get values for the encrypted cookie tests. Flipped back